lms/static/sass/course/wiki/_wiki.scss

```diff
@@ -746,7 +751,7 @@
   }
 
   .filter-clear {
-    margin-right: ($baseline/2);
+    margin-left: ($baseline/2);
```

Comment:
Yes, the wiki has a host of RTL issues. That's beyond the scope of this PR (and given the general level of "unsupport" of the wiki, I don't think it is something we intend to fix).